### PR TITLE
ZEB-1391/ fix modify test action according permissions on ui

### PR DESCRIPTION
**Pull request by MitskevichVS**

### CODE REVIEW

File: client/app/containers/test-details/test-details.html [hunk 1]
```diff
@@ -109,8 +109,9 @@
                             ng-repeat="btn in $ctrl.batchButtons"
                             flex="none"
                             class="{{  btn.class }}"
-                            ng-class="{ '_completed': btn.completed }"
-                            ng-click="btn.onClick($event,  btn)">
+                            ng-class="{ '_completed': btn.completed, '_disabled': !$ctrl.userHasAnyPermission(['MODIFY_TESTS']) }"
+                            ng-click="btn.onClick($event,  btn)"
+                            ng-disabled="!$ctrl.userHasAnyPermission(['MODIFY_TESTS'])">
```
Comment (danymill) [inline client/app/containers/test-details/test-details.html:114]: I think you don't need `_disabled` class, you can use valid `[disabled]` selector in the CSS

File: client/styles/layout/test-details.scss [hunk 2]
```diff
@@ -686,6 +686,28 @@
                                     }
                                 }
                             }
+                            &._disabled {
```
Comment (danymill) [inline client/styles/layout/test-details.scss:689]: See comment above

**Review by danymill — CHANGES_REQUESTED** [review]

File: client/app/containers/test-details/test-details.html (hunk 1 above)
Comment (MitskevichVS) [reply client/app/containers/test-details/test-details.html:114]: fixed

File: client/styles/layout/test-details.scss (hunk 2 above)
Comment (MitskevichVS) [reply client/styles/layout/test-details.scss]: fixed

File: client/styles/layout/test-details.scss [hunk 3]
```diff
@@ -687,6 +687,29 @@
...
+                                        border-color: $green-new;
+                                    }
+                                }
+                                &._red-icon {
+                                    .btn-icon {
+                                        background-color: $red-new;
+                                        border-color: $red-new;
+                                    }
+                                }
+                            }
```
Comment (danymill) [inline client/styles/layout/test-details.scss:711]: code duplication & it doesn't code `:focus`, for example

**Review by danymill — CHANGES_REQUESTED** [review]

File: client/styles/layout/test-details.scss (hunk 3 above)
Comment (MitskevichVS) [reply client/styles/layout/test-details.scss]: fixed

**Review by danymill — APPROVED** [review]

**Pull request merged**